modules/phase_field/examples/grain_growth/grain_growth_3D.i

```diff
@@ -9,42 +9,48 @@
   dim = 3 # Problem dimension
   nx = 10 # Number of elements in the x-direction
   ny = 10 # Number of elements in the y-direction
-  nz = 3
+  nz = 10
   xmin = 0    # minimum x-coordinate of the mesh
   xmax = 1000 # maximum x-coordinate of the mesh
   ymin = 0    # minimum y-coordinate of the mesh
   ymax = 1000 # maximum y-coordinate of the mesh
   zmin = 0
-  zmax = 300
-  uniform_refine = 2 # Initial uniform refinement of the mesh
+  zmax = 1000
+  uniform_refine = 1 # Initial uniform refinement of the mesh
 
-  parallel_type = replicated # Periodic BCs
+  parallel_type = distributed
 []
 
 [GlobalParams]
   # Parameters used by several kernels that are defined globally to simplify input file
-  op_num = 9 # Number of order parameters used
+  op_num = 15 # Number of order parameters used
```

Comment:
Right, now that the initial condition has been fixed - it's seeing a lot more conflicts. It turns out that for the 25 grains, we now need 15 order parameters instead of 9 like I was getting away with before. The upshot is that we don't have any merging grains on the first timestep!